magenta/models/shared/melody_rnn_generate.py

```diff
@@ -198,7 +202,7 @@ def run(melody_encoder_decoder, build_graph):
                       FLAGS.primer_midi)
 
   run_generate(graph, train_dir, FLAGS.output_dir, melody_encoder_decoder,
-               primer_melody, FLAGS.num_steps, bpm)
+               primer_melody, FLAGS.num_steps, FLAGS.bpm)
```

Comment:
We should be careful here. Right now, the bpm is derived from either the BPM present in the priming sequence or a default of 120. This change would remove the ability to base the BPM on the BPM present in the priming sequence.

If we have a flag, I'd prefer for it to take effect up on line 185. It could be referenced instead of melodies_lib.DEFAULT_BEATS_PER_MINUTE. That way, if a BPM is present in the priming sequence, it will be used, but if there isn't a BPM specified, the flag value will be used.